test/runtests.jl

```diff
@@ -133,6 +133,13 @@ end
     histogram([1, 0, 0, 0, 0, 0])
 end
 
+@testset "Callable Functions" begin
+    struct MyFun end
```

Comment:
Travis complains about this - " type definition not allowed inside a local scope"